### PR TITLE
Rename workers.dev to Cloudflare Workers

### PR DESCRIPTION
**Pull request by jerrychong25**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -756,7 +756,7 @@ Table of Contents
   * [scn.sap.com](https://scn.sap.com/docs/DOC-56411) — The in-memory Platform-as-a-Service offering from SAP. Free developer accounts come with 1 GB structured, 1 GB unstructured, 1 GB of Git data and allow you to run HTML5, Java and HANA XS apps
   * [staroid.com](https://staroid.com) - Managed Kubernetes namespace service designed to fund open source developers. Free 8 CPUs and 16GB of RAM namespace to test branches and pull requests of public repository. Free test namespace shutdown every 30 minutes. Maximum 2 concurrent test namespaces.
   * [SUSE Developer Program](https://developer.suse.com) — Experience cloud native productivity for free. Get hands-on with the SUSE Cloud Application Platform with your own Developer Sandbox. 1 Free Application. Free subdomain provided along with API for CLI. Storage & Memory Quota of 1 GB.
-  * [workers.dev](https://workers.dev) - Deploy serverless code for free on Cloudflare's global network. 100,000 free requests per day with a workers.dev subdomain.
+  * [Cloudflare Workers](https://workers.cloudflare.com/) - Deploy serverless code for free on Cloudflare's global network. 100,000 free requests per day with a workers.dev subdomain.
```
Comment (ripienaar) [inline README.md:759]: maybe move it to the cloud flare entry then that lists all their features?

Comment (jerrychong25) [reply README.md:759]: Do you mean move this to “**Major Cloud Providers**' Always-Free Limits” section?

FYI. Currently there is another Cloudflare listing under “**CDN and Protection**” section.

Comment (ripienaar) [reply README.md:759]: Under `CDN and Protection` yes, put this there with the existing one

Comment (jerrychong25) [reply README.md]: Done moved to existing Cloudflare listing. Can review again =)

**Comment by ripienaar** [comment]
thanks! :)

**Comment by jerrychong25** [comment]
> thanks! :)

@ripienaar No problem!

**Pull request merged**